### DISCUSSION TITLE
Where to find GitHub icons/symbols package?

### DISCUSSION THREAD

**Discussion by irvnriir** [community/community#24691] [Apps, API and Webhooks]
<p>All icons it use, not only the GitHub product/organization icon . And what license they are under ? (if its not stated there)</p>

**Comment by yakov116** [comment, marked as answer]
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/primer/octicons">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/primer/octicons" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/16872793/181086271-79932782-2b2d-4ae8-9b43-30a31e9b006a.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/765375d735686b2931343eb78319dc9493d916b9_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/6/765375d735686b2931343eb78319dc9493d916b9_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/6/765375d735686b2931343eb78319dc9493d916b9.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/765375d735686b2931343eb78319dc9493d916b9_2_10x10.png"></div>

<h3><a href="https://github.com/primer/octicons" target="_blank" rel="noopener nofollow ugc">GitHub - primer/octicons: A scalable set of icons handcrafted with &lt;3 by...</a></h3>

  <p>A scalable set of icons handcrafted with &lt;3 by GitHub - GitHub - primer/octicons: A scalable set of icons handcrafted with &lt;3 by GitHub</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
